### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by dominik-korsa**

### CODE REVIEW

File: commands/ping.js [hunk 1]
```diff
@@ -2,9 +2,12 @@ const Discord = require('discord.js');
 
 exports.run = async (client, message) => {
   const testmessage = await message.channel.send('Sprawdzam...');
-  const embed = new Discord.RichEmbed()
+  const embed = new Discord.MessageEmbed()
     .setTitle('PONG! :ping_pong:')
-    .addField(`Udało mi się odpowiedzieć w ${Math.round(testmessage.createdTimestamp - message.createdTimestamp)}ms`, `Opóźnienie API wynosi ${Math.round(client.ping)}ms`)
```
Comment (doteq) [inline commands/ping.js]: Why you changed that? `.addField` is still working on v12

Comment (dominik-korsa) [reply commands/ping.js]: Initially I removed Discord ping support as it wasn't working. When I figured out how to fix this I didn't go back to addField as it's not the intended use and the styling in new Discord update makes it too distinct.

Comment (doteq) [reply commands/ping.js]: What was not working? For me, addField in this case looks better, because api ping is less important than message ping.

Comment (dominik-korsa) [reply commands/ping.js:7]: image
Well, currently the name is grayed out, and the value is white. I don't think the name/value should be used for styling either, as the importance is only your opinion.

Comment (doteq) [reply commands/ping.js:7]: Oh ok. As I remember, earlier values of embed fields are grayed out, not names.

**Pull request merged**